modules/common/src/main/scala/notebook/front/widgets/Generic.scala

```diff
@@ -0,0 +1,34 @@
+package notebook.front.widgets
+
+import scala.xml.{NodeSeq, UnprefixedAttribute, Null}
+import play.api.libs.json._
+import notebook._
+import notebook.front._
+import notebook.JsonCodec._
+import notebook.front.widgets.magic
```

Comment:
Here and elsewhere, it would be more idiomatic to import `magic` and `magic._` that the full name.